flash/core/data/utils.py

```diff
@@ -40,7 +40,6 @@
     RunningStage.PREDICTING: "predict",
     RunningStage.SERVING: "serve",
 }
-_STAGES_PREFIX_VALUES = {"train", "test", "val", "predict", "serve"}
```

Comment:
this is not used anywhere. alternative is `_STAGES_PREFIX_VALUES  = set(_STAGES_PREFIX.values())`